apps/api_web/lib/api_web/legacy_stops.ex

```diff
@@ -0,0 +1,279 @@
+defmodule ApiWeb.LegacyStops do
+  @moduledoc """
+  Enables backwards compatibility for changes to stop IDs and "splitting" of stops, e.g. from a
+  station having one non-platform-specific child stop to having a child stop for each platform.
+
+  Mappings from old to new stop IDs are expressed as:
+
+    %{"2020-01-01" => %{"old_id" => {"new_id", ["new_1", "new_2"]}}}
+
+  In this example, stop "old_id" was renamed to "new_id" and two sibling stops were added (if no
+  rename or additions occurred, these elements would be `nil` and `[]` respectively). Requests
+  involving "old_id" using API versions earlier than 2020-01-01 should be treated as though they
+  are *also* for "new_id", and possibly "new_1" and "new_2" as well. Retaining the old ID allows
+  requests to work both before and after the ID changes occur in GTFS.
+
+  An "old" ID may appear in multiple versions. Chained mappings where one version's "new" ID is a
+  later version's "old" ID are also possible.
+  """
+
+  alias Model.Stop
+
+  @mappings %{
+    "2018-07-23" => %{
+      "Back Bay" =>
+        {nil, ["Back Bay-01", "Back Bay-02", "Back Bay-03", "Back Bay-05", "Back Bay-07"]},
+      "North Station" =>
+        {nil,
+         [
+           "North Station-01",
+           "North Station-02",
+           "North Station-03",
+           "North Station-04",
+           "North Station-05",
+           "North Station-06",
+           "North Station-07",
+           "North Station-08",
+           "North Station-09",
+           "North Station-10"
+         ]},
+      "South Station" =>
+        {nil,
+         [
+           "South Station-01",
+           "South Station-02",
+           "South Station-03",
+           "South Station-04",
+           "South Station-05",
+           "South Station-06",
+           "South Station-07",
+           "South Station-08",
+           "South Station-09",
+           "South Station-10",
+           "South Station-11",
+           "South Station-12",
+           "South Station-13"
+         ]}
+    },
+    "2019-02-12" => %{
+      "70001" => {nil, ["Forest Hills-01", "Forest Hills-02"]},
+      "70036" => {nil, ["Oak Grove-01", "Oak Grove-02"]},
+      "70061" => {nil, ["Alewife-01", "Alewife-02"]},
+      "70105" => {nil, ["Braintree-01", "Braintree-02"]},
+      "70150" => {nil, ["71150"]},
+      "70151" => {nil, ["71151"]},
+      "70200" => {nil, ["71199"]},
+      "70201" => {nil, ["Government Center-Brattle"]},
+      "70202" => {nil, ["Government Center-Brattle"]}
+    },
+    "2020-XX-XX" => %{
+      "Abington" => {"PB-0194-S", []},
+      "Anderson/ Woburn" => {"NHRML-0127", ["NHRML-0127-01", "NHRML-0127-02"]},
+      "Andover" => {"WR-0228-02", []},
```

Comment:
@jfabi For cases like this, I thought it was appropriate for it to be a "rename" since the old stop ID is in fact going away and there is only one new stop ID that replaces it. The only functional difference is that it allows e.g. `/stops/Andover` to keep working in older versions. Does this seem right to you?

Reply:
Yep, this sounds right to me. 🎉